frontend/shared/components/src/events/EventOverview.vue

```diff
@@ -177,52 +175,64 @@ const props = defineProps<{
 const title = computed(() => props.event.name);
 const $navigate = useNavigate();
 const organization = useOrganization();
-const context = useContext()
-const platform = usePlatform()
+const context = useContext();
+const platform = usePlatform();
 const pop = usePop();
-const groupOrganization = ref<Organization | null>(null);
+const groupOrganization: Ref<Organization | null> = ref(null);
 
 function setOrganization(o: Organization) {
-    groupOrganization.value = o as any;
+    groupOrganization.value = o;
 }
 
-const hasFullAccess = computed(() => {
-    return !organization.value || props.event.organizationId === organization.value.id;
+const canAdminEvent = computed(() => {
+    if (props.event.organizationId) {
+        if (props.event.organizationId === organization.value?.id) {
+            return canAdminEventHelper(organization.value);
+        }
+        else if (groupOrganization.value) {
+            return canAdminEventHelper(groupOrganization.value);
+        }
+        return false;
+    }
+
+    return canAdminEventHelper(null);
```

Comment:
```suggestion
return canAdminEventHelper(groupOrganization.value);
```
Dit doet net hetzelfde. groupOrganizatiov is enkel null als de event oragnizationId ook null is + het gebruikt ook de context organization